integrationTests/mock/contracts/errors.go

```diff
@@ -0,0 +1,9 @@
+package contracts
+
+import "errors"
+
+// ErrNilTokenHandler signals that a nil token handler instance has been provided
+var ErrNilTokenHandler = errors.New("nil TokenHandler instance")
+
+// ErrInsufficientArguments signals that insufficient arguments were provided
+var ErrInsufficientArguments = errors.New("insufficient argument")
```

Comment:
`insufficient arguments` . also, as stated in the previous comment, I would name it `ErrInvalidNumberOfArguments` or something similar